docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md

```diff
@@ -0,0 +1,308 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Get your website or web application online by setting up Apache, MariaDB, and PHP'
```

Comment:
Should we add "on Debian 10" and "LAMP stack" to the description to help for SEO?

Reply:
Updated description...fixed